scripts/rollup/cleanup-plugin.mjs

```diff
@@ -37,7 +38,9 @@ function cleanup() {
           babelrc: false
         });
       } else if (dtsFilter(chunk.fileName)) {
-        return code.replace(emptyImportRe, '');
+        return code
+          .replace(emptyImportRe, '')
+          .replace(gqlImportRe, x => '/*!@ts-ignore*/\n' + x);
```

Comment:
**NOTE:** This build change marks `graphql` imports in out `d.ts` typings files with a `/*!@ts-ignore*/` pragma. While imports from `graphql` will change to `any` when the package isn’t installed, we still need to mark this in case a user has `skipLibCheck: false` set in their tsconfig